tempesta_fw/cache.c

```diff
@@ -636,6 +636,15 @@ tfw_cache_send_304(TfwHttpReq *req, TfwCacheEntry *ce)
 	WARN_ON_ONCE(!list_empty(&req->fwd_list));
 	WARN_ON_ONCE(!list_empty(&req->nip_list));
 
+	if (TFW_MSG_H2(req)) {
+		/*
+		 * TODO: add separate flow for HTTP/2 response preparing
```

Comment:
Please mark such TODOs as `TODO #309` that it'll be easy to follow your TODOs and for reviewers.

Reply:
Done.